docs/source/release/v6.14.0/mantidworkbench.rst

```diff
@@ -7,7 +7,13 @@ Mantid Workbench Changes
 
 New Features
 ------------
-.. amalgamate:: Workbench/New_features
+- The ability to scale axis and colorbars to ``symlog`` has been added to plots. An option to specify the extent of the linear portion of the ``symlog`` axis (``linthresh``) has been added to the axes tab widget in the plot config dialog.
+- Sequential fit output workspace names now include the numeric (spectra) axis to distinguish between different results.
+- Sequential fit interface now has an option to create the results output in a workspace 2D format which could be plotted.
+- Created :ref:`PythonObjectProperty <PythonObjectProperty>` which allows using generic python objects as inputs to algorithms defined through the python API
+- Created :ref:`PythonObjectTypeValidator <PythonObjectTypeValidator>` for use with :ref:`PythonObjectProperty <PythonObjectProperty>`, to enforce correct typing on python objects
+- The crosshair button is now enabled in tiled plots(subplots).
+- The crosshair has now been optimized using matplotlib's blitting function for fast rendering. It can very closely tracks mouse movement with minimal lag now.
```

Comment:
```suggestion
- The crosshair has now been optimized using matplotlib's blitting function for fast rendering. It can very closely track mouse movement with minimal lag now.
```